prime-dotnet-webapi/Services/SiteService.cs

```diff
@@ -295,6 +295,19 @@ public async Task<int> UpdateCompletedAsync(int siteId)
             return updated;
         }
 
+        public async Task<Site> UpdateSiteAdjudicator(Site site, Admin admin = null)
+        {
+            site.Adjudicator = admin;
+
+            var updated = await _context.SaveChangesAsync();
+            if (updated < 1)
+            {
+                throw new InvalidOperationException($"Could not update the site adjudicator.");
```

Comment:
Pretty sure this will throw if you called this twice in a row (the site's Adjudicator isnt changed in the second call and so no objects are updated).  Is that intended behavior?

Reply:
Removed markism